src/main/java/de/tum/in/www1/artemis/domain/iris/message/IrisMessage.java

```diff
@@ -19,38 +19,44 @@
 
 /**
  * An IrisMessage represents a single message in an IrisSession.
- * A message can be created by either the user, an LLM, or Artemis.
- * Artemis messages are used to give commands to the bot and are not displayed to the user.
+ * The message may contain multiple pieces of content with different types.
  */
 @Entity
 @Table(name = "iris_message")
 @Cache(usage = CacheConcurrencyStrategy.NONSTRICT_READ_WRITE)
 @JsonInclude(JsonInclude.Include.NON_EMPTY)
 public class IrisMessage extends DomainObject {
-
+    
     @ManyToOne
     @JsonIgnore
     private IrisSession session;
-
+    
     @Nullable
     @Column(name = "sent_at")
-    private ZonedDateTime sentAt;
-
+    private ZonedDateTime sentAt = ZonedDateTime.now();
+    
     @Nullable
     @Column(name = "helpful")
     private Boolean helpful;
-
+    
     @Column(name = "sender")
     @Enumerated(EnumType.STRING)
     private IrisMessageSender sender;
-
-    @Transient
-    private Integer messageDifferentiator; // is supposed to be only a part of the dto and helps the client application to differentiate messages it should add to the message store
-
+    
     @OrderColumn(name = "iris_message_content_order")
     @OneToMany(mappedBy = "message", fetch = FetchType.LAZY, cascade = CascadeType.ALL, orphanRemoval = true)
     private List<IrisMessageContent> content = new ArrayList<>();
-
+    
+    @Transient
+    private Integer messageDifferentiator; // is supposed to be only a part of the dto and helps the client application to differentiate messages it should add to the message store
+    
+    // Required by JPA
+    public IrisMessage() {}
+    
+    public IrisMessage(IrisSession session) {
```

Comment:
Why do you need this specific constructor now?
For new messages the session must be set in a specific way, so this does not make much sense, outside maybe the tests.